src/environment/test-schema-version.ts

```diff
@@ -1 +1 @@
-export const version = '3.2.4';
+export const version = '3.2.5';
```

Comment:
Was this necessary as part of this hotfix?

Reply:
I didn't change this. somehow it changed itself. Believe Carl is doing something to remove this from the repository and git ignore it.